.github/actions/environment/action.yml

```diff
@@ -53,7 +53,7 @@ runs:
           maui-android \
           ${{ runner.os == 'macOS' && 'maui-ios maui-maccatalyst maui-windows maui-tizen' || '' }} \
           ${{ runner.os == 'Windows' && 'maui-windows' || '' }} \
-          --temp-dir "${{ runner.temp }}"
+          --temp-dir "${{ runner.temp }}" --from-rollback-file rollback.json
```

Comment:
I didn't even know the [option existed](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-restore) 😮

Reply:
It's not in the public documentation.  Lots of GitHub issues refer to it though, as does this:
https://github.com/dotnet/sdk/blob/main/documentation/general/workloads/workload-rollback.md